### PR TITLE
Updated local linkage instructions

### PR DESCRIPTION
**Pull request by maneesht**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,14 +190,16 @@ scope](https://www.npmjs.com/search?q=scope%3Afirebase) on NPM.
 
 ### Testing the SDK Locally
 
-Please be sure to build your repo before proceeding any further.
+Please be sure to build your SDK changes and install your dependencies before proceeding any further.
```
Comment (IanWyszynski) [inline README.md:193]: Maybe just say, "run `yarn build` in all packages under test"?

Comment (maneesht) [reply README.md]: `all packages under test` isn't quite correct. I updated it. PTAL

**Review by IanWyszynski — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -190,14 +190,16 @@ scope](https://www.npmjs.com/search?q=scope%3Afirebase) on NPM.
...
 
 ```shell
 $ cd packages/firebase
 $ yarn link # initialize the linking to the other folder
-$ cd ../<my-test-app-dir> # cd into your personal project directory
-$ yarn link firebase # tell yarn to use the locally built firebase SDK instead
+$ cd ../packages/<my-product> # Example: $ cd packages/database
+$ yarn link # link your product to make it available elsewhere
+$ cd <my-test-app-dir> # cd into your personal project directory
+$ yarn link firebase firebase/<my-product> # tell yarn to use the locally built firebase SDK instead
```
Comment (hsubox76) [inline README.md:202]: Does this work? I thought it was the package name, e.g. `@firebase/<my-product>`

Comment (maneesht) [reply README.md:202]: Ah, nice catch.

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**